src/usart/state.rs

```diff
@@ -32,6 +32,16 @@ impl Word for u16 {
     }
 }
 
+/// Indicates that a USART instance is operation in asynchronous mode
```

Comment:
```suggestion
/// Indicates that a USART instance is operating in asynchronous mode
```

Reply:
Thanks. I could literally not see what the difference between my version and what you were suggesting here in the notification email. Good thing the web interface does the colored highlighting to help my tired brain out a bit.